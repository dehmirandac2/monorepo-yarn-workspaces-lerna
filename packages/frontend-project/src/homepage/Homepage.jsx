import React from "react";
import Button from "@clients/components/lib/button";

const Homepage = () => (
  <h1>
    My Homepage teste
    <Button />
  </h1>
);
export default Homepage;
