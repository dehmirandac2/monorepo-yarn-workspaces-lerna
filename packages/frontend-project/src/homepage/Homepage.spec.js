import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";
import Homepage from ".";

describe("Homepage", () => {
  test("renders correctly", () => {
    const tree = renderer.create(<Homepage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
