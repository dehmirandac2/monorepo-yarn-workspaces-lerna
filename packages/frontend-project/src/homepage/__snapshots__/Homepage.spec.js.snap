// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage renders correctly 1`] = `
<h1>
  My Homepage teste
  <button
    className=""
  />
</h1>
`;
